organizers/2017-07-18.html

```diff
@@ -0,0 +1,86 @@
+---
+layout: organizers
+---
+
+<section>
+    <h1>The London Django Meetup Group</h1>
+    <h2>18th July 2017</h2>
+</section>
+
+<section data-background='#eee'>
+    <h1>Sponsorship</h1>
+    <p>Tonight's meetup is kindly sponsored by:</p>
+    <ul>
+        <li><strong>Festicket</strong></li>
+        <li><strong>Time Out</strong></li>
+        <li><strong>Funding Options</strong></li>
```

Comment:
Might be better to sort in alphabetical order now to indicate no preference (and following slides)